test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingFromHeaderTest.cs

```diff
@@ -93,7 +93,7 @@ public class ModelBindingFromHeaderTest
             var result = JsonConvert.DeserializeObject<Result>(body);
             Assert.Equal<string>(tags, result.HeaderValues);
             var error = Assert.Single(result.ModelStateErrors);
-            Assert.Equal("Title", error);
+            Assert.Equal("BlogTitle", error);
```

Comment:
because the validation changed, the validation now uses what is specified in the name and not the property name.